src/components/Navigator.vue

```diff
@@ -203,8 +209,22 @@ export default {
 }
 
 .loading-placeholder {
+  align-items: center;
   color: var(--color-figure-gray-secondary);
-  padding: 12px;
-  @include font-styles(body-reduced);
+  justify-content: center;
+}
+
+.loading-spinner {
+  --spinner-size: 40px; // used for both width and height
+  --spinner-delay: 1s; // don't show spinner until this much time has passed
+
+  height: var(--spinner-size);
+  visibility: visible;
```

Comment:
`visibility: visible;` is the default value. I don't think we need to specify it, right?

Reply:
https://github.com/apple/swift-docc-render/pull/244/commits/fdbea73c9a062e2ae868f1426bd750d199f6c6c4